docs/src/components/components/icon-ui/README.md

```diff
@@ -1,3 +1,24 @@
 # Icon Ui
 
-Needs documentation...
+The **icon ui** component is used to assist the user interface to help the user
```

Comment:
`used to assist the user interface to help the user` sounds a little confusing. Maybe just `used to assist the user`?